v0.10.7 blog post (#45)
endless-sky/endless-sky.github.io: _posts/2024-05-25-unstable-release-0.10.7.md

Have you ever wondered why cloaking paints your ship dark red, and all of a sudden nobody in the galaxy can see it? Well, thanks to Koranir, not only does your ship turn red, but a fancier outline for cloaking effects makes it partially transparent! This can be toggled within the settings.

<img class="centered shadowed" src="/images/blog/v0.10.7/cloak.png" width="400" height="225" />

While we're talking about new settings, Koranir has also introduced a new "Dynamic Camera Acceleration" setting that makes flying feel a bit more dynamic, with the camera panning away from your ship based on its acceleration. This is currently defaulted to on, but it can also be turned off in the settings. We may change it to default to off based on feedback.

A neat new mechanic turrets can now be given is a "firing arc", where you can define the start and end angles that a turret hardpoint can rotate, as well as what direction it faces. This can be useful for if you want to make turrets that don't shoot through large structures on a ship model, or to give a ship blindspots that it can't fire turrets from 1010todd and oo13.

> On the content side, missions can now "mark" systems of importance without the player being required to visit them, as well as leave them marked after being visited. Not only that, but mission triggers and conversations can do various new things such as play music, give the player debt, and perform events immediately instead of having to wait at least one day. @Amazinite

Reviewer: ```suggestion
On the content side, missions can now "mark" systems of importance without the player being required to visit them, as well as leave them marked after being visited. Not only that, but mission triggers and conversations can do various new things such as play music, give the player debt, and perform events immediately instead of having to wait at least one day, thanks to Amazinite.
```

> Around human space, if you keep your eyes peeled, you will find a few more planets and stations that weren't there last time, bringing with them a little of fluff. Some of these can be used as stopovers for content that otherwise didn't have a nearby planet to land at. Thanks to @DoomKorath and @bene_dictator for contributing these.

Reviewer: ```suggestion
Around human space, if you keep your eyes peeled, you will find a few more planets and stations that weren't there last time, bringing with them a little of fluff. Some of these can be used as stopovers for content that otherwise didn't have a nearby planet to land at. Thanks to DoomKorath and bene-dictator for contributing these.
```

<img class="centered shadowed" src="/images/blog/v0.10.7/uninhabited.png" width="400" height="225" />

> If you're itching for more missions, DoomKorath has also introduced a short mission series for the Wanderers involving an interaction with the Mereti. What for, you'll have to see for yourself! There are a few other new short mission chains coming with this update aside from this one, so a decent amount of new short story content to get involved with, with the help bene_dictator, spearsdane, Saugia, and several others.

Reviewer: ```suggestion
If you're itching for more missions, DoomKorath has also introduced a short mission series for the Wanderers involving an interaction with the Mereti. What for, you'll have to see for yourself! There are a few other new short mission chains coming with this update aside from this one, so a decent amount of new short story content to get involved with, with the help bene-dictator, spearsdane, Saugia, and several others.
```

A friendly reminder that in order to play unstable updates on Steam or GOG, you have to opt into the beta branch.

- On Steam, go to your library, right click Endless Sky, go to Properties, go to Betas, select the beta in the drop down box, and that's it! Also a reminder that we now have a GOG release, and you can find that here if you're interested. 

- On GOG, go to your library, right click Endless Sky, go to Manage Installation -> Configure, and select the beta in the Beta Channels drop down box.

If you're using the High DPI plugin, you should also update that to the [current version](https://github.com/endless-sky/endless-sky-high-dpi/releases/v0.10.7).

Next release will be v0.10.8 Stable, meaning it will focus on bug fixes for this release, and have a shorter cycle ending somewhere around the end of June. This won't require you to opt into the beta branch once available.

> If you'd like to give feedback on anything else this update, or on the game in general, here is our feedback box. https://forms.gle/6A9kEcrFHSLw9CQw5 With all of the meaty changes above, there is plenty to talk about. Play around with the new update for a bit, then head over to the survey and give us your thoughts!

Reviewer: ```suggestion
If you'd like to give feedback on anything else this update, or on the game in general, here is our [feedback box](https://forms.gle/6A9kEcrFHSLw9CQw5). With all of the meaty changes above, there is plenty to talk about. Play around with the new update for a bit, then head over to the survey and give us your thoughts!
```

> The Github discussion post that comes with this release can be found here. (LINK)

Reviewer: bruh

Author: ```suggestion
The Github discussion post that comes with this release can be found [here](https://github.com/endless-sky/endless-sky/discussions/10105).
```

> And as always, a reminder to star us on Github if you haven't yet to help the game's visibility!

Reviewer: ```suggestion
And as always, a reminder to star us on [Github](https://github.com/endless-sky/endless-sky) if you haven't yet to help the game's visibility!
```
